hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/bootstrap/HoodieSparkBootstrapSchemaProvider.java

```diff
@@ -92,7 +92,7 @@ private static Schema getBootstrapSourceSchemaOrc(HoodieWriteConfig writeConfig,
     try {
       orcReader = OrcFile.createReader(filePath, OrcFile.readerOptions(context.getHadoopConf().get()));
     } catch (IOException e) {
-      throw new HoodieException("Could not determine schema from the data files.");
+      throw new HoodieException("Could not determine schema from the data files. Create orc reader error.");
     }
```

Comment:
`Could not determine schema from the ORC data files.`

Reply:
@danny0405 thanks，danny.  i have update this exception message